src/mono/wasm/README.md

```diff
@@ -4,16 +4,19 @@ This depends on `emsdk` to be installed.
 
 ## emsdk
 
-* You can either install it yourself (https://emscripten.org/docs/getting_started/downloads.html), and set `EMSDK_PATH` to that. Make sure to have this set whenever building, or running tests for wasm.
-
-* Or you can run `make provision-wasm`, which will install it to `$reporoot/src/mono/wasm/emsdk`.
+* You can run `make provision-wasm`, which will install it to `$reporoot/src/mono/wasm/emsdk`.
 Note: Irrespective of `$(EMSDK_PATH)`'s value, `provision-wasm` will always install into `$reporoot/src/mono/wasm/emsdk`.
 
 `EMSDK_PATH` is set to `$reporoot/src/mono/wasm/emsdk` by default, by the Makefile.
 
 Note: `EMSDK_PATH` is set by default in `src/mono/wasm/Makefile`, so building targets from that will have it set. But you might need to set it manually if
 you are directly using the `dotnet build`, or `build.sh`.
 
+* Alternatively you can install [expected version](./emscripten-version.txt) yourself from the [Emscripten SDK guide](https://emscripten.org/docs/getting_started/downloads.html).
+Do not install `latest` but rather specific version e.g. `./emsdk install 2.0.12`
+
+Make sure to set `EMSDK_PATH` variable, whenever building, or running tests for wasm.
```

Comment:
+1, it used to have a default value of `$reporoot/src/mono/wasm/emsdk`, but not anymore.

Reply:
I'm not sure what do you mean, could you please elaborate ?

Reply:
Before `make -C src/mono/wasm provision-wasm; ./build.sh -os browser` used to work (without explicitly setting `EMSDK_PATH`) as the provision script installs it in a default (predictable) location. It doesn't work anymore; we have to set `EMSDK_PATH` in all cases.

Reply:
I think we are in the middle of transition to use nuget packages for emscripten and  I think this change is related to that. @radical knows the details.